### PR TITLE
Bump Root to v6-28-02

### PR DESCRIPTION
**Pull request by pzhristov**

### CODE REVIEW

File: root.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package: ROOT
 version: "%(tag_basename)s"
-tag: "v6-26-10-alice5"
-source: https://github.com/alisw/root.git
```
Comment (sawenzel) [inline root.sh:4]: I think that we will loose a couple of commits in our fork which have not been backported to root. This should not be done.

**Review by sawenzel — CHANGES_REQUESTED** [review]
I think that we will loose a couple of commits in our fork which have not been backported to root. This should not be done.

**Review by ktf — APPROVED** [review]

**Comment by ktf** [comment]
Merging this to see if we spot other issues in production. We can have a alice1 tag with the missing treatment of ROOT_LDSYSPATH. Are we sure this is still relevant, though?

**Comment by sawenzel** [comment]
> Merging this to see if we spot other issues in production. We can have a alice1 tag with the missing treatment of ROOT_LDSYSPATH. Are we sure this is still relevant, though?

Yes, I have no reason to believe otherwise.

**Pull request merged**